src/bundle/ui-dev/src/modules/common/thumbnail/thumbnail.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-
 import Icon from '../icon/icon';
 
-const Thumbnail = ({ thumbnailData, iconExtraClasses, contentTypeIconPath }) => {
+interface ThumbnailProps {
+    thumbnailData: {
+        mimeType: string;
+        resource: string;
+    };
+    iconExtraClasses?: string;
+    contentTypeIconPath?: string | null;
+}
+
+const Thumbnail = ({ thumbnailData, iconExtraClasses = '', contentTypeIconPath = null }: ThumbnailProps) => {
```

Comment:
Do we need `''` as default for `iconExtraClasses`?
```suggestion
const Thumbnail = ({ thumbnailData, iconExtraClasses, contentTypeIconPath = null }: ThumbnailProps) => {
```